servers/cromwell/jobs/__main__.py

```diff
@@ -31,13 +33,42 @@
     args, _ = parser.parse_known_args()
 
 app = connexion.App(__name__, specification_dir='./swagger/', swagger_ui=False)
+DEFAULT_CROMWELL_CREDENTIALS = {"cromwell_user": "", "cromwell_password": ""}
+LOADER_ERR_MSG = "Failed to load config.json, using the default config: {0}".format(
+    DEFAULT_CROMWELL_CREDENTIALS)
+
+# Load credentials for cromwell
 config_path = os.environ.get('CROMWELL_CREDENTIALS')
-with open(config_path) as f:
-    config = json.load(f)
+
+# Check if the credentials are provided properly
+try:
+    with open(config_path) as f:
+        config = json.load(f)
+except (IOError, TypeError):
```

Comment:
If there is a config.json, I think we want to check if it contains the `"cromwell_user"` and `"cromwell_password" keys, otherwise this would still throw an error when trying to check the credentials later on.

Reply:
That's a good catch! I should have caught this... will do!